runtime/src/weights/pallet_scheduler.rs

```diff
@@ -36,9 +36,9 @@ impl<T: frame_system::Config> pallet_scheduler::weights::WeightInfo for WeightIn
     // Storage: Preimage StatusFor (r:1 w:1)
     // Storage: Scheduler Lookup (r:0 w:1)
     fn on_initialize_periodic_named_resolved(s: u32) -> Weight {
-        (36_758_000 as Weight)
-            // Standard Error: 313_000
-            .saturating_add((48_916_000 as Weight).saturating_mul(s as Weight))
+        (0 as Weight)
```

Comment:
Is this as expected? I checked, and this is non-zero in the `weights.rs` of the scheduler pallet 4.0.0.

Reply:
This is out of my control, sometimes the benchmarks for the same benchmark logic start with a base weight of greater than zero and sometimes it's zero. I think as long as the weight is modified properly by variables this is acceptable. If a change in the logic did not happen we can revert the base weight tough, I think it's better to overcharge than to undercharge (undercharging can lead to overweight blocks that can't be produced in time).